backend/order/models.py

```diff
@@ -43,3 +44,20 @@ class OrderItem(models.Model):
 
     def __str__(self):
         return f'{self.listing} : {self.quantity}'
+
+
+class FeedbackAndRating(models.Model):
+    given_to = models.ForeignKey(User, on_delete=models.CASCADE, related_name='given_to')
+    given_from = models.ForeignKey(User, on_delete=models.CASCADE, related_name='given_from')
+    related_order = models.ForeignKey(Order, on_delete=models.CASCADE, related_name='related_order')
+    rating = models.PositiveIntegerField(
+        default=1,
+        choices=[(i, i) for i in range(1, 6)],
```

Comment:
@julkascript If you mean this line with choices I think the expression should not cause any error since in the migration it is transfromed to numbers.